### PR TITLE
Update report-junk-email-and-phishing-scams-in-outlook-for-iOS-and-An…

### PR DESCRIPTION
**Pull request by andypunt**
…droid.md

We should remove or update this article to use the Report Message or Report Phishing add-in experience. The native reporting experience has issues and we should no longer point people to it for security flows.

### CODE REVIEW

**Comment by siosulli** [comment]
Hi @andypunt just before I merge this. In the Report Message and Report Phish articles we state: 

"If you're an admin in a Microsoft 365 organization with Exchange Online mailboxes, we recommend that you use the Submissions portal in the Security & Compliance Center." 

so I was just wondering if this new text is a bit conflicting? 

We could put a note at the top of this article instead, to state something like: 

To report messages to Microsoft, we recommend using the Report Message add-in or the Report Phishing add-in.

**Comment by andypunt** [comment]
> Hi @andypunt just before I merge this. In the Report Message and Report Phish articles we state:
> 
> "If you're an admin in a Microsoft 365 organization with Exchange Online mailboxes, we recommend that you use the Submissions portal in the Security & Compliance Center."
> 
> so I was just wondering if this new text is a bit conflicting?
> 
> We could put a note at the top of this article instead, to state something like:
> 
> To report messages to Microsoft, we recommend using the Report Message add-in or the Report Phishing add-in.

That is still applicable since admin submissions is more powerful than user submissions. You can update the statement to call that out if needed. In other words, to report messages, you should use the Report Message or Phishing add-ins, but if you are an admin, you can use admin submissions in the security portal because it is gives you more in terms of feedback.

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/report-junk-email-and-phishing-scams-in-outlook-for-iOS-and-Android.md [hunk 1]
```diff
@@ -40,32 +40,4 @@ In Microsoft 365 organizations with mailboxes in Exchange Online or on-premises
...
-
-1. Select one or more messages.
-2. In the top-right corner tap on the three vertical dots. The action menu opens.
-
-   ![Report not junk email from the action menu](../../media/Android-not-junk-email.png)
-
-3. Tap **Not junk**.
-
-A toast notification appears that the email has moved to your Inbox. If you change your mind, select **Undo** on the toast notification. The email remains in the Junk folder.
+Admins should use the Report Message or the Report Phishing add-in experience to report messages: https://docs.microsoft.com/microsoft-365/security/office-365-security/enable-the-report-message-add-in
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/report-junk-email-and-phishing-scams-in-outlook-for-iOS-and-Android.md]:
```suggestion
Admins should use the Report Message or the Report Phishing add-in experience to report messages. See [Enable the Report Message add-in](https://docs.microsoft.com/microsoft-365/security/office-365-security/enable-the-report-message-add-in) for more information.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@andypunt

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks.

**Comment by siosulli** [comment]
@yogkumgit I will merge this morning. I just have one update to make after a discussion with Andy. thanks!

**Pull request merged**